tools/docker-run.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+set -e -o pipefail
+
+# Common error handling
+bail() {
+    if [[ $# -gt 0 ]]; then
+        >&2 echo "Error: $*"
+    fi
+    exit 1
+}
+
+SCRIPT_PATH="$1"
+
+# Get SDK image
+source="$(tq -r ".sdk.source" -f Twoliter.lock)"
+version="$(tq -r ".sdk.version" -f Twoliter.lock)"
+_name="$(tq -r ".sdk.name" -f Twoliter.lock)"
+_registry="${source%/*}"
+
+registry="$(tq -r ".bottlerocket.bottlerocket-sdk.registry" -f Twoliter.override 2>/dev/null || echo "$_registry")"
+name="$(tq -r ".bottlerocket.bottlerocket-sdk.name" -f Twoliter.override 2>/dev/null || echo "$_name")"
+
+SDK="${registry}/${name}:v${version}"
+
+# Run the script in docker
+docker run --rm \
+    -v "$(pwd):/bottlerocket-kernel-kit" \
+    --user "$(id -u):$(id -g)" \
+    --name "kernel-full-config" \
```

Comment:
You don't have to give this a name, if anything, it shouldn't be always `kernel-full-config` as the purpose of the script is to be used to run any script within a docker container